src/Bigcommerce/Api/Connection.php

```diff
@@ -537,6 +544,11 @@ private function parseHeader($curl, $headers)
         if (!$this->responseStatusLine && strpos($headers, 'HTTP/') === 0) {
             $this->responseStatusLine = $headers;
         } else {
+            $trimHeader = trim($headers);
+            if (!empty($trimHeader)) {
+                $this->responseHeadersList[] = $trimHeader;
```

Comment:
🤔 do we want to classify them by key? something like:

```php
$this->responseHeaderList[$key][] = $value;
```

Reply:
We could. What I plan to do with them is iterate through looking for the `Set-Cookie` that has the `XSRF-TOKEN`. I suppose that is a little more performant if we have already filtered by the `$key`.

Do you think it is worth the extra complication here?

Reply:
I don't mind either way really 🤷

Reply:
Ya that looks good to me. Makes my parsing code a little easier to read in regards to searching the `Set-Cookie` headers. I think it would make it a little more annoying if reading other headers that only have a single entry. This is how it looks:
<img width="961" alt="Screenshot 2025-02-25 at 9 42 08 AM" src="https://github.com/user-attachments/assets/df1490ce-4209-4ce9-8d26-95b31d3ee49c" />

Reply:
Good point. You could and if by some chance there is multiple then it would be available here.

Reply:
Updated pr. 👍

Reply:
Typing was griefing me but got it going now 😅